guide/daily-spending-wallet/requesting.md

```diff
@@ -34,440 +34,400 @@ Illustration source
-
-If there is no amount defined on the invoice, the invoice is denominated in bitcoin, or the invoice needs to be shared in a message, then use a longer expiry, like 24 hours.
-
-While it is good to allow users to define their own custom expiries, remember that only advanced users will do this. Choose a sensible default that makes sense for your user-base and how the wallet is intended to be used.
-
-For some background on situations the user might encounter with different invoice expiries, see this [blog post](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/) from designer Stephen DeLorme.
-
-Using [BOLT12](https://bolt12.org/) offers, invoice expirations will no longer be a point of user friction. Offers are not yet an accepted standard, or widely supported by wallets, so interoperability will be an issue if adopting it.
-
-</div>
-
-### Address types
-
-For on-chain, there are several address types that can be used for single-use payment requests.
-
-<div class="center" markdown="1">
-
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings-address.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings-address@2x.png"
-   layout = "float-right-desktop -background -shadow"
-   caption = "Supporting every address type benefits users and makes your wallet more interoperable."
-   alt-text = "Screen showing different address types."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Invoice.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Invoice@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Invoice@2x.png"
+   layout = "shadow"
+   caption = "Let users share the lightning invoice independently."
+   alt-text = "Screen showing an invoice that is part of a unified request in QR format."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
-Each address type has unique benefits, such as SegWit saving on network fees and Taproot offering Lightning users increased privacy.
-
-You should encourage users to use the latest, more feature rich address types. Having an option in the payment request settings to set which address type to be used by default is recommended.
-
-Older address types should always be supported for backwards compatibility with older wallets.
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/requesting/address.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/address@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/address@2x.png"
+   layout = "shadow"
+   caption = "Let users share the on-chain address independently."
```

Comment:
Since this on-chain address is a swap address, the screen can indicate the extra fee.